src/test/java/seedu/address/testutil/EventBuilder.java

```diff
@@ -48,7 +48,14 @@ public EventBuilder withDescription(String description) {
         return this;
     }
 
+    /**
+     * Builds an Event with the attributes from EventBuilder
+     */
     public Event build() {
+        if (time == null) {
+            return new Event(date, description);
+        }
```

Comment:
I don't think you need to handle this case anymore as I've removed the `time` param from the `requireAllNonNull` check in `Event`'s constructor